<launch>
<group ns="map_merge">
  <node pkg="map_merge_3d" type="map_merge_node" respawn="false" name="map_merge_3d" output="screen">
    <param name="robot_map_topic" value="map"/>
    <param name="robot_namespace" value=""/>
    <param name="merged_map_topic" value="map"/>
    <param name="world_frame" value="world"/>
    <param name="compositing_rate" value="0.3"/>
    <param name="discovery_rate" value="0.05"/>
    <param name="estimation_rate" value="0.01"/>
    <param name="publish_tf" value="true"/>
    <!-- and all the other map merging parameters -->
  </node>
</group>
</launch>
